src/client.ts

```diff
@@ -71,6 +69,15 @@ enum TransactionStatus {
   TRANS_UNKNOWN = 4, // cannot determine status
 }
 
+class TimeoutError extends Error {
```

Comment:
I don't like this -- ideally this should be (a) exported, (b) defined in the `errors/` module.  The latter is a bit complicated, since it's autogenerated.  I suggest to simply use the standard `Error` class for timeout errors (as do 99% of all NodeJS packages).

Reply:
OK, I've removed `TimeoutError` in favor of `Error`.